gifs/build.gradle.kts

```diff
@@ -0,0 +1,97 @@
+import java.nio.file.Path
+import java.nio.file.Paths
+
+group = rootProject.group
+version = rootProject.version
+
+val finalTaskNames = listOf("FinishTheGame", "FinishGame", "CreateTheApp", "FinishTheApp", "CompleteTheProject")
```

Comment:
You can probably rename the last step in each lesson to "CompleteTheProject", since you refactored the whole course; what do you think?